### PR TITLE
Wrote comments on vars

### PR DESCRIPTION
**Pull request by Pajarogers**
Made hitboxes red

### CODE REVIEW

File: objects/oplayercontroller/Create_0.gml [hunk 1]
```diff
@@ -1,11 +1,13 @@
 /// @description Insert description here
 // You can write your code in this editor
 
+// Determines which player port this object is using (Player 1, Player 2, etc)
 playerID = 1;
+
 // Movment Variables
-hsp = 0;
+hsp = 0; // Horizontal speed
```
Comment (johnkiddjr) [inline objects/oplayercontroller/Create_0.gml:8]: Name this better, please.

File: objects/oplayercontroller/Create_0.gml [hunk 2]
```diff
@@ -81,11 +82,13 @@ CharacterSprites = {
 	knockdown_Sprite : sRussel_KnockedDown,
 }
 
+// State-related Variables
```
Comment (johnkiddjr) [inline objects/oplayercontroller/Create_0.gml:85]: This comment is useless.

**Review by johnkiddjr — CHANGES_REQUESTED** [review]
Please see my line by line comments

File: objects/oplayercontroller/Create_0.gml (hunk 1 above)
Comment (johnkiddjr) [reply objects/oplayercontroller/Create_0.gml:8]: Ok this looks good now!

**Review by johnkiddjr — APPROVED** [review]

**Comment by Pajarogers** [comment]
ok

**Pull request closed without merging**